_data/code/federal/statcan.yml

```diff
@@ -28,3 +28,46 @@ releases:
         - Data
       fr:
         - Données
+  - contact:
+      email: noreply@canada.ca
+      name: 'https://github.com/drupalwxt/wxt/issues/new'
```

Comment:
Issue tracker new page doesn't seem right for a name, but I think I get what you were trying to do